CommonTools/ParticleFlow/python/EITopPAG_EventContent_cff.py

```diff
@@ -15,7 +15,7 @@
     'keep recoPFTaus_pfTausEI_*_*',
     #'keep recoPFTauDiscriminator_pfTausDiscrimination*_*_*',
     'keep recoPFTauDiscriminator_pfTausDiscriminationByDecayModeFinding_*_*',
-    'keep recoPFTauDiscriminator_pfTausDiscriminationByIsolation_*_*',
+    'keep recoTauDiscriminatorContainer_pfTausDiscriminationByIsolation_*_*',
```

Comment:
this did not work.
The class name is ```recoSingleTauDiscriminatorContaineredmValueMap_```
This full "friendly" name or some wildcard like ```'keep *Discriminator*_pfTausDiscriminationByIsolation_*_*',``` is needed (assuming that this is the only product).